Jenkinsfile

```diff
@@ -236,6 +237,7 @@ pipeline {
                         sh 'cd build && cmake -DCMAKE_BUILD_TYPE=Release -DTERRIER_USE_ASAN=OFF -DTERRIER_BUILD_BENCHMARKS=OFF .. && make -j4'
                         sh 'cd build && gtimeout 1h make unittest'
                         sh 'cd build && gtimeout 1h make check-tpl'
+                        // sh 'cd build && gtimeout 10s python3 ../script/testing/kill_server_on_port.py 15721'
```

Comment:
Are we still doing the experiment where we will make sure that we kill the DBMS in some environments for this stage and not others? If not, then I think you need to make sure that we kill the server before we call `make unittest` at every stage

Reply:
`kill_server_on_port.py` is now called before every `make unittest`